converters/google/convert.go

```diff
@@ -249,10 +250,15 @@ func (c *Converter) augmentAsset(tfData converter.TerraformResourceData, cfg *co
 // projects. It implements a cache because many resources share the same
 // project.
 func (c *Converter) getAncestry(project string) (string, error) {
+	log.Printf("getAncestry: project=%s", project)
```

Comment:
won't this be too spammy? can we only print this when `--verbose` is set?

Reply:
There is a log suppression logic in root.go when --verbose is set:

```
		if !flags.verbose {
			// Suppress chatty packages.
			log.SetOutput(ioutil.Discard)
		}
```